www/src/templates/template-docs-markdown.js

```diff
@@ -44,87 +45,88 @@ function DocsTemplate({ data, location, pageContext: { next, prev } }) {
         <meta name="twitter.label1" content="Reading time" />
         <meta name="twitter:data1" content={`${page.timeToRead} min read`} />
       </Helmet>
-      <Layout
-        location={location}
-        locale={page.fields.locale}
-        enableScrollSync={urlSegment === `docs` ? false : true}
-      >
-        <DocSearchContent>
-          <Container
-            overrideCSS={{
-              pb: 0,
-              [mediaQueries.lg]: {
-                pt: 9,
-              },
-              [toc && mediaQueries.xl]: {
-                ...containerStyles,
-              },
-            }}
-          >
-            <Breadcrumb location={location} />
-            <h1 id={page.fields.anchor} sx={{ mt: 0 }}>
-              {page.frontmatter.title}
-            </h1>
-          </Container>
-          <Container
-            overrideCSS={{
-              pt: 0,
-              position: `static`,
-              [mediaQueries.lg]: {
-                pb: 9,
-              },
-              [toc && mediaQueries.xl]: {
-                ...containerStyles,
-                display: `flex`,
-                alignItems: `flex-start`,
-              },
-            }}
-          >
-            {toc && (
+      <Layout location={location} locale={page.fields.locale}>
+        <PageWithSidebar
+          location={location}
+          enableScrollSync={urlSegment === "docs" ? false : true}
```

Comment:
only the tutorial actually uses this, so we could change this to `urlSegment === "tutorial"` possibly

Reply:
What sections make use of that template? `docs`, `tutorial` and `contributing`? If so I agree with changing it explicitly to `tutorial`